pkg/osquery/table/chrome_login_keychain_emails.go

```diff
@@ -0,0 +1,140 @@
+package table
+
+import (
+	"context"
+	"database/sql"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/pkg/errors"
+
+	"github.com/kolide/kit/fs"
+	"github.com/kolide/osquery-go"
+	"github.com/kolide/osquery-go/plugin/table"
+)
+
+var profileDirs = map[string][]string{
+	"windows": []string{"Appdata/Local/Google/Chrome/User Data"},
+	"darwin":  []string{"Library/Application Support/Google/Chrome"},
+	"default": []string{".config/google-chrome", ".config/chromium", "snap/chromium/current/.config/chromium"},
+}
+
+func ChromeLoginKeychainEmails(client *osquery.ExtensionManagerClient, logger log.Logger) *table.Plugin {
+	c := &ChromeLoginKeychainEmailsTable{
+		client: client,
+		logger: logger,
+	}
+	columns := []table.ColumnDefinition{
+		table.TextColumn("username"),
+		table.TextColumn("email"),
+		table.BigIntColumn("count"),
+	}
+	return table.NewPlugin("kolide_chrome_login_keychain_emails", columns, c.generate)
+}
+
+type ChromeLoginKeychainEmailsTable struct {
+	client *osquery.ExtensionManagerClient
+	logger log.Logger
+}
+
+func (c *ChromeLoginKeychainEmailsTable) generateForPath(ctx context.Context, path string, username string) ([]map[string]string, error) {
+	dir, err := ioutil.TempDir("", "kolide_chrome_login_keychain_emails")
+	if err != nil {
+		return nil, err
+	}
+	defer os.RemoveAll(dir) // clean up
+
+	dst := filepath.Join(dir, "tmpfile")
+	if err := fs.CopyFile(path, dst); err != nil {
+		return nil, err
+	}
+
+	db, err := sql.Open("sqlite3", dst)
+	if err != nil {
+		return nil, err
+	}
+	defer db.Close()
+
+	db.Exec("PRAGMA journal_mode=WAL;")
```

Comment:
perhaps a comment on why we need this would be warranted here?

Reply:
This was carried over from the chrome_login_keychain.go I will do some digging.